particles/readfields.h

```diff
@@ -119,20 +119,37 @@ std::vector<double> readFieldData(Reader& r, std::string& name, unsigned int num
       exit(1);
    }
 
-   if(dataType != vlsv::datatype::type::FLOAT || byteSize != 8 || vectorSize != numcomponents) {
```

Comment:
Shouldn't there still be a check for:
```
if(dataType != vlsv::datatype::type::FLOAT || vectorSize != numcomponents) {

Reply:
We *could* hypothetically plot the technical grid? The particle pusher might, for example, extract boundary information from there?